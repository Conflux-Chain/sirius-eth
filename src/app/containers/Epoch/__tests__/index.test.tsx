import React from 'react';
import { render } from '@testing-library/react';

const renderComponent = () => render(<div>Todo. Epochs</div>);

describe('<Epochs />', () => {
  it('should match the snapshot', () => {
    const component = renderComponent();
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
