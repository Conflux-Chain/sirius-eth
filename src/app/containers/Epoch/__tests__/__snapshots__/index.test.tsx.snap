// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Epochs /> should match the snapshot 1`] = `
<div>
  Todo. Epochs
</div>
`;
